CHANGELOG.md

```diff
@@ -5,32 +5,38 @@ Notable changes to this project are documented in this file. The format is based
 ## [Unreleased]
 
 Breaking changes (😱!!!):
```

Comment:
Did we decide to go emoji-free for changelogs?

Reply:
We're going emoji-free for the core libraries, but it's up to us to decide whether we want to go emoji-free for `contrib` too. I'm happy either way; do you have an opinion, @JordanMartinez?

Reply:
Let's keep in in the Contributors libraries :)